### DISCUSSION TITLE
Missing Click on a Button (Avalonia UI)

### DISCUSSION THREAD

**Discussion by giziben** [AvaloniaUI/Avalonia#10621] [Q&A] [answered]
Hi, as experienced WPF programmer I was surprised there was no **Click** on my Button.
e.g. 
<Button **Click**="DrawCircleButton_Click" Height="40" Width="120" Content="Draw Circle" </Button>

**Comment by maxkatz6** [comment, marked as answer]
There is a [Click event](https://github.com/AvaloniaUI/Avalonia/blob/master/src/Avalonia.Controls/Button.cs#L118) on a Button. Please check your error messages.

**Reply by giziben** [reply]
Thanks a lot, I was used to get the code behind for me as in WPF!

**Reply by timunie** [reply]
Rider does it for you, but VisualStudio unfortunately not.

**Comment by giziben** [comment]
I am too tied to VS, thanks.
